src/index.ts

```diff
@@ -83,24 +84,38 @@ program
         if (majorVersion) {
             newMajorVersion();
         } else {
-            const categories = await getCategoryNames();
+            const breaking = "Yes, breaking change";
```

Comment:
I don't think this prompt for breaking change should be implemented here. Right now, `publish` is used to create a major release or to publish the block for the first time. This question should be make when running `release` in my opinion.

`Are you *releasing* a breaking change?`
yes -> Inform the user to create a major release (using `publish -m`, but don't do it automatically for them.
no -> Continue

The reason for not doing it automatically is that the user need to decide how they will handle the new version of the block (a new branch, a copy of the folder, etc)

Reply:
Sounds fair. I was just implementing what was described, if you think this is better I'll make the change.